_posts/2021-04-01-openttd-1-11-0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: OpenTTD 1.11.0
+author: TrueBrain
+---
+
+Another year, another release.
+Who would have guessed ;)
+
+With 1.11 we bring you a lot of new features which you hopefully enjoy as much as we did creating them:
+* Performance improvements - Fast-Forward is now so fast, there is a setting to rate limit it (on by default).
```

Comment:
Worth mentioning OpenGL/"hardware acceleration" explicitly?

Reply:
I wanted to mention a lot of things; but I couldn't find a funny way of wording it all. So I settled for this by the lack of better. We don't have to mention all the details, I guess.

Also, without giving more context, I was somewhat afraid people would think the game now has a 3D engine or something silly :P

But, if you have a single sentence to mention something like that, which isn't all too serious, please do make a suggestion :D

Reply:
"In part due to a new hardware-accelerated video renderer, fast-forward is now..." ?

Reply:
That would make a really long sentence. I think we just leave it for what it is now. The changelogs gives them the full story, I guess :P